r/content/reading_and_writing_data.Rmd

```diff
@@ -304,6 +304,39 @@ test_that("read_json_arrow chunk works as expected", {
 unlink(tf)
 ```
 
+### Discussion
```

Comment:
Thanks for submitting the PR. The content is great, though I'm not entirely convinced it belongs in the Cookbook.

FWIW I think this might be going into too much detail regarding working with JSON imported a list-columns in a dplyr-tidyr workflow. We could say something along the lines of ... once you import / read the data in, you can then work with it in several ways: tidyr or purrr workflows, etc. IMHO working with nested structures is slightly out-of-scope for the cookbook. We could link to the working with nested data tidyr [vignette](https://tidyr.tidyverse.org/articles/nest.html) or Jenny Bryan's working with [list-columns in purrr](https://jennybc.github.io/purrr-tutorial/ls13_list-columns.html).

Reply:
Agreed - there's a really delicate balance to be struck here between "explaining how to do things using the arrow package" (desirable) and "explaining how to do things using the tidyverse in general" (undesirable).  We've come up against this in the past, when [writing the recipe for using dplyr verbs in arrow](https://arrow.apache.org/cookbook/r/manipulating-data---tables.html#use-dplyr-verbs-in-arrow), and making a conscious choice _not_ to detail each of the main dplyr verbs that we support (as this is dplyr content and not arrow content).

I think you're right @eitsupi that it could be helpful to point readers in the right direction here but agree with @dragosmg that this level of detail is a little out-of-scope and that a brief sentence with a link would be best in this particular case.

Reply:
Thank you both for your review. I agree with your remarks and close this.